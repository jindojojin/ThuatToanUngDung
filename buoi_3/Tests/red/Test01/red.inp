30 6
1 1 6 2 2 5 2 1 3 3 1 3 1 6 1 2 6 3 5 2 5 6 5 2 1 2 3 2 5 2 
15 5 27 9 24 30 
