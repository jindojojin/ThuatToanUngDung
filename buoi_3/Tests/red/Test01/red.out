5 1
1 7
4 0
0 4
5 3
8 0
